### DISCUSSION TITLE
Corro los test y me aparecen incorrectos

### DISCUSSION THREAD

**Discussion by bauticorrea** [atralice/Curso.Prep.Henry#197] [Q&A]
Están las correcciones bien hechas y me sigue diciendo que hay 34 failed.
![unknown_2022 03 14-15 59](https://user-images.githubusercontent.com/95256560/158243162-279b9434-c41b-440b-903e-a3da060b7a03.png)

**Comment by MarianaStocco** [comment]
Hola! Asegurate de guardar los cambios que hagas antes de correr el test, con Ctrl+S... Me pasó al principio lo mismo, capaz que es eso. Un saludo!
